Tests/ApodiniTests/ModifierTests/ConcatenatedResponseTransformerTests.swift

```diff
@@ -52,10 +52,20 @@ final class ConcatenatedResponseTransformerTests: ApodiniTests {
                     .response(SecondTestResponseMediator())
             }
         }
+
+        let webService = TestWebService()
+        let restExporter = RESTInterfaceExporter(app)
+        webService.configuration.configure(app)
+
+        webService.register(
+            SharedSemanticModelBuilder(app)
+                .with(exporter: restExporter)
+                .with(exporter: GRPCInterfaceExporter.self),
+            GraphQLSemanticModelBuilder(app),
+            WebSocketSemanticModelBuilder(app)
+        )
         
-        TestWebService.main(app: app)
-        
-        try app.test(.GET, "/v1/") { res in
+        try restExporter.app.test(.GET, "/v1/") { res in
```

Comment:
I use `Vapor.Application.test` myself and I really like its functionality. However, this doesn't look encouraging for new tests.
We should probably create some kind of `Apodini.Application.test`, but I wouldn't require it for this PR. I can try implementing it myself :)

Reply:
well the test is rest specific anyway so this is just fine. For other exporter we will have to add similar tests.

Reply:
I just decided to use one shared vapor application behind the scenes so this is no longer an issue